src/BloomBrowserUI/bookEdit/js/editViewFrame.js

```diff
@@ -153,10 +153,18 @@ function showAddPageDialog(templatesJSON) {
     var theDialog;
     var parentElement = document.getElementById('page').contentWindow;
 
+    // don't show if a dialog already exists
+    if ($(document).find(".ui-dialog").length) {
+        return;
+    }
     parentElement.localizationManager.loadStrings(getAddPageDialogLocalizedStrings(), null, function() {
 
         var title = parentElement.localizationManager.getText('AddPageDialog.Title', 'Add Page...');
-        var dialogContents = CreateAddPageDiv(templatesJSON);
+        var addButtonText = parentElement.localizationManager.getText('AddPageDialog.AddPageButton', 'Add This Page');
```

Comment:
These localization things came in when I rebased... don't know why github is considering them part of this PR